This isn't building on EXZ:

/home/h01/david.davies/cylc-run/monio-29/share/mo-bundle/monio/src/monio/Utils.cc: In function 'std::string monio::utils::exec(const std::string&)':
/home/h01/david.davies/cylc-run/monio-29/share/mo-bundle/monio/src/monio/Utils.cc:64:25: error: aggregate 'std::array<char, 128> buffer' has incomplete type and cannot be defined
   64 |   std::array<char, 128> buffer;
      |                         ^~~~~~
gmake[2]: *** [monio/src/CMakeFiles/monio.dir/build.make:300: monio/src/CMakeFiles/monio.dir/monio/Utils.cc.o] Error 1

--- a/src/monio/Utils.cc
+++ b/src/monio/Utils.cc
@@ -64,9 +64,7 @@ bool strToBool(std::string input) {
 std::string exec(const std::string& cmd) {
   std::array<char, 128> buffer;
   std::string result;
-
   std::string extCmd = "sh -c '" + cmd + "' 2>&1";
-
   std::unique_ptr<FILE, decltype(&pclose)> pipe(popen(extCmd.c_str(), "r"), pclose);
   if (!pipe) {
     throw std::runtime_error("popen() failed!");

--- a/src/monio/Utils.cc
+++ b/src/monio/Utils.cc
@@ -11,6 +11,7 @@
 #include <stdio.h>
 
 #include <algorithm>
+#include <array>
 #include <fstream>
 #include <memory>
 

